### PR TITLE
Add log monitoring

### PR DESCRIPTION
**Pull request by lethjakman**

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -262,13 +272,13 @@ func ParseConfig(fn string) (Configuration, error) {
...
 	}
 
-    val, ok = input.Get("bridge", "rabbit_mq_auto_delete_queue")
-    if ok {
-        b, err := strconv.ParseBool(val)
-        if err == nil {
-            config.AMQPAutoDeleteQueue = b
-        }
-    }
+	val, ok = input.Get("bridge", "rabbit_mq_auto_delete_queue")
```
Comment (brianebeyer) [inline config.go]: please undo these changes, they are not part of the PR

Comment (lethjakman) [reply config.go]: Yeah, that was just goformat. Lemme see what I can do about that.

File: main.go [hunk 2]
```diff
@@ -380,6 +382,32 @@ func startOutputs() error {
...
 }
 
+func monitorLog(logToMonitor string) {
+	fmt.Printf("monitoring log %s", logToMonitor)
+	t, _ := tail.TailFile(logToMonitor, tail.Config{Follow: true})
+	for line := range t.Lines {
+		var logJson map[string]interface{}
+		logJson = make(map[string]interface{})
+		logJson["message"] = line.Text
+		logJson["type"] = "log"
```
Comment (brianebeyer) [inline main.go:392]: need to add "filename"

Comment (lethjakman) [reply main.go:392]: Below
